tests/Aspire.Pomelo.EntityFrameworkCore.MySql.Tests/AspireEFMySqlExtensionsTests.cs

```diff
@@ -2,28 +2,38 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using Aspire.Components.Common.Tests;
+using Aspire.Hosting.MySql;
+using Aspire.MySqlConnector.Tests;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore.Infrastructure.Internal;
 using Microsoft.EntityFrameworkCore.Storage;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using MySqlConnector;
 using Pomelo.EntityFrameworkCore.MySql.Infrastructure.Internal;
 using Xunit;
 
 namespace Aspire.Pomelo.EntityFrameworkCore.MySql.Tests;
 
-public class AspireEFMySqlExtensionsTests
+public class AspireEFMySqlExtensionsTests : IClassFixture<MySqlContainerFixture>
 {
-    private const string ConnectionString = "Server=localhost;User ID=root;Database=test";
     private const string ConnectionStringSuffixAddedByPomelo = ";Allow User Variables=True;Use Affected Rows=False";
+    private readonly MySqlContainerFixture _containerFixture;
+    private string ConnectionString => RequiresDockerTheoryAttribute.IsSupported
+                                            ? _containerFixture.GetConnectionString()
+                                            : "Server=localhost;User ID=root;Password=pass;Database=test";
+    private readonly string _serverVersion = $"{MySqlContainerImageTags.Tag}-{MySqlContainerImageTags.Image}";
```

Comment:
```suggestion
    private readonly string _serverVersion = $"{MySqlContainerImageTags.Tag}-mysql";
```

This isn't the Image name. Yes they happen to be the same, but it's not what this value is. There are different flavors of MySql databases (ex. MariaDB). This string tells the EF library which kind of database to prepare statements for.